File: ccc/2014/j3.py
#https://dmoj.ca/problem/ccc14j3
#CCC '14 J3 - Double Dice

r = int(input())
a = 0
d = 0
while r > 0:
    y = input()
    x = list(y)
    for i in x:
        if i == " ":
            x.remove(i)
    r -= 1
    if x[0] > x[1]:
        d += int(x[0])
    elif x[0] < x[1]:
        a += int(x[1])

print(100-a)
print(100-d)

File: ccc/2007/j3.java
//https://dmoj.ca/problem/ccc07j3
//CCC '07 J3 - Deal or No Deal Calculator

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		int[] amounts = {100, 500, 1000, 5000, 10000, 25000, 50000, 100000, 500000, 1000000};
		int n = s.nextInt();
		for (int i = 0; i < n; i++) {
			int a = s.nextInt();
			amounts[a - 1] = 0;
		}
		int num = s.nextInt();
		int sum = 0;
		int counter = 10;

		for (int i = 0; i < amounts.length; i++) {
			sum += amounts[i];
			if (amounts[i] == 0) {
				counter--;
			}
		}

		num *= counter;
		if (num >= sum) {
			System.out.println("deal");
		} else {
			System.out.println("no deal");
		}
		s.close();
	}
}
